### ISSUE TITLE
Error: spawn EACCES

### ISSUE THREAD

**Issue by chxii** [desktop/desktop#5304]
## Description

I just downloaded Github Desktop, and got this error message when I opened it: spawn EACCES
<img width="415" alt="screen shot 2018-07-31 at 15 42 50" src="https://user-images.githubusercontent.com/16949423/43491555-e1b64f88-94d9-11e8-920e-0996ace0cc49.png">
I could not close this Error message to use Github Desktop.

## Version

* GitHub Desktop: 1.3.1

* Operating system: Mac OS 10.13.6

## Steps to Reproduce

Open Github Desktop.

## Additional Information

New to this and not sure if I did something wrong. Please tell me if more info is needed. 

### Logs

[2018-07-31.desktop.production.log](https://github.com/desktop/desktop/files/2247356/2018-07-31.desktop.production.log)

**Comment by shiftkey** [comment]
@chxii thanks for the report, and log file!

This looks like the two relevant errors you are seeing:

```
2018-07-31T22:58:31.301Z - warn: [ui] Error refreshing pull requests for 'polyreg'
Error: spawn EACCES
    at _errnoException (util.js:1024:11)
    at ChildProcess.spawn (internal/child_process.js:323:11)
    at exports.spawn (child_process.js:514:9)
    at Object.exports.execFile (child_process.js:224:15)
    at Object.module.(anonymous function) [as execFile] (ELECTRON_ASAR.js:199:20)
    at /webpack:/app/node_modules/dugite/build/lib/git-process.js:66:1
    at new Promise (<anonymous>)
    at Function.exec (/webpack:/app/node_modules/dugite/build/lib/git-process.js:49:1)
    at a.measure (/webpack:/app/src/lib/git/core.ts:119:3)
    at Object.n [as measure] (/webpack:/app/src/ui/lib/git-perf.ts:27:4)
```

**Could the folder permissions have changed for your `polyreg` repository recently?** I've not seen any recent reports of this error code, and I'd like to reproduce this issue on my end.

```
2018-07-31T22:58:38.617Z - error: [ui] Error performing periodic fetch
Error: spawn EACCES
    at _errnoException (util.js:1024:11)
    at ChildProcess.spawn (internal/child_process.js:323:11)
    at exports.spawn (child_process.js:514:9)
    at Object.exports.execFile (child_process.js:224:15)
    at Object.module.(anonymous function) [as execFile] (ELECTRON_ASAR.js:199:20)
    at /webpack:/app/node_modules/dugite/build/lib/git-process.js:66:1
    at new Promise (<anonymous>)
    at Function.exec (/webpack:/app/node_modules/dugite/build/lib/git-process.js:49:1)
    at a.measure (/webpack:/app/src/lib/git/core.ts:119:3)
    at Object.n [as measure] (/webpack:/app/src/ui/lib/git-perf.ts:27:4)
```

This error is coming from the background fetcher, and is likely the same issue for the same repository, but we don't have that identifying information from the first error.

**Comment by chxii** [comment]
My problem was solved in Version 1.3.2. Yes, I just found that folder doesn't exist in my new Mac. Thanks a lot for your quick response and great help!

**Comment by billygriffin** [comment]
@chxii Great! Glad it got resolved, thanks again for the report!
